doc/source/examples/dem/granuheap/granuheap.rst

```diff
@@ -0,0 +1,396 @@
+- Moving solid surface
+- `GMSH <https://gmsh.info/>`_ grids
+- Multi-case simulations
+
+
+----------------------------
+Files Used in This Example
+----------------------------
+
+All files mentioned below are located in the example's folder (``examples/dem/3d-granuheap``).
+
+- Mesh file: ``cylinder.msh`` and ``support.msh``
+- Python file to generate mesh files: ``gmsh.py``
+- Parameters files for the problem: ``granuheap.prm`` and ``granuheap_multicase.prm``
+- Post-processing python files: ``post_processing.py``, ``post_processing_multicase.prm`` and ``post_processing_functions.py``.
+
+-----------------------
+Description of the Case
+-----------------------
+
+This example simulates the flow of wet sand during a Granuheap experiment. First, the packing of particles is generated by inserting particles inside a cylinder. After 1.6s, the cylinder rises, and the particles can flow freely, producing a heap. We compare the numerical results with our experimental results. To remain consistent with the Granuheap experiment, we use the same metrics in our simulations: a cylinder of 0.01m in diameter. For more information aboutt the Granuheap procedures, visit the `Granuheap web page <https://www.granutools.com/en/granuheap>`_. To match our experimental results, we test multiple combinations of parameters in our simulations. We use the work of Hu *et al.* [#hu2022]_ as a basis to choose which contact properties will create those combinations.
+
+-------------------
+Generate Mesh Files
+-------------------
+
+To generate the ``cylinder.msh`` and ``support.msh`` files, we used the ``gmsh.py`` file. The two functions take as input the bottom's position (x0), the radius, the number of points in the circumference, and the height. 
+
+.. code-block:: text
+
+  cylinder(x0, radius, n_points, height)
+  support(x0, radius, n_points, height)
+
+
+In this case, the cylinder is 0.02 tall with a radius of 0.005 and the support is 0.005 tall and has the same diameter as the cylinder. We fix the number of points in the circumference at 50 for both the cylinder and the support. 
+
+.. code-block:: text
+
+  cylinder(-0.005, 0.005, 50, 0.02)
+  support(-0.01 ,0.005 ,50, 0.005)
+
+.. figure:: images/meshfiles.png
+    :width: 400
+    :alt: Mesh
+    :align: center
+
+Mesh files can be generated using the following command.
+
+.. code-block:: text
+  :class: copy-button
+
+  python3 gmsh.py
+
+.. note::
+  GMSH is needed to generate mesh files. See `GMSH website <https://gmsh.info/>`_ for installation and `Introduction on How to Use GMSH <../../../tools/gmsh/gmsh.html>`_ for more information. 
+
+
+--------------
+Parameter File
+--------------
+
+Mesh
+~~~~~
+
+The domain we simulate is a rectangular box measuring :math:`0.045\times0.02\times0.02` meters, made using the deal.ii grid generator.  The grid is refined 6 times using the ``set initial refinement`` parameter so that the cell size is approximately 1.5 times the highest diameter value.
```

Comment:
```suggestion
The domain we simulate is a rectangular box measuring :math:`0.045\times0.02\times0.02` meters, made using the deal.ii grid generator.  The grid is refined 6 times using the ``set initial refinement`` parameter so that the cell size is approximately 1.5 times the largest particle diameter value.
```